core/src/BibleEngine.class.ts

```diff
@@ -145,28 +144,22 @@ export class BibleEngine {
         this.pDB = this.createConnection(dbConfig);
     }
 
-    async findConnection(dbConfig: ConnectionOptions) {
-        try {
-            return getConnection(dbConfig.name || this.CONNECTION_NAME).manager;
-        } catch (error: any) {
-            if (error?.name === 'ConnectionNotFoundError') {
-                return this.createConnection(dbConfig);
-            }
-            throw error;
-        }
+    async findConnection(dbConfig: DataSourceOptions) {
```

Comment:
nit: can annotate a return type for clarity (yes, TS will automatically validate based on the return type whether its manually annotated or not, but its just helpful for readability)

Reply:
You mean specifying a return type via TS or jsdoc? 
Are you saying that each method should be annotated manually? Or more like in cases where its not obvious and/or you want to enforce a certain type in the implementation (that's how I use it currently)? I can see the point about readability. On the other hand you do work that TS can do for you. Do you know about the inlay hints feature in vscode? It shows you the type of things in-line.

Reply:
Also, you mentioned in slack that we don't need this method anymore? Then we should better remove it altogether? Less code = better